python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -638,6 +649,56 @@ def _find_stars(self, exposure, background):
         else:
             return result.sourceCat
 
+    def _match_psf_stars(self, psf_stars, stars):
+        """Match calibration stars to psf stars, to identify which were psf
+        candidates, and which were used or reserved during psf measurement.
+
+        Parameters
+        ----------
+        psf_stars : `lsst.afw.table.SourceCatalog`
+            Psf candidate stars that were sent to the psf determiner. Used to
```

Comment:
PSF candidates maybe?  Technically it should be `PSF` everywhere, but we also use `psf`.  And yet the title-case `Psf` is awkward.